rancher2/resource_rancher2_machine_config_v2.go

```diff
@@ -47,8 +47,21 @@ func resourceRancher2MachineConfigV2Create(d *schema.ResourceData, meta interfac
 		MinTimeout: 3 * time.Second,
 	}
 	_, waitErr := stateConf.WaitForState()
```

Comment:
Can we get rid of this call and just do the retry? Maybe increase the retry to 4 and move this into the for-loop.